README.md

```diff
@@ -1 +1,11 @@
 # Data Structures & Algorithms in Go
+
+## Installing Go
```

Comment:
`Installing Go` should be a section in `Requirements`. For example:

## Requirements

- Go [v1.0.0+](https://golang.org/doc/install)